fedocal/templates/default/master.html

```diff
@@ -98,6 +98,7 @@ <h4 class="hide">{{ _('Main Site Links') }}</h4>
             filename='jquery-ui.min-1.11.1.js') }}">
     </script>
     <script type="text/javascript">
+        FEDOCAL = {};
```

Comment:
What about putting this one in ``base_partial`` and call ``{{ super() }}`` in the template inheriting from it